src/app/create/page.tsx

```diff
@@ -37,44 +44,111 @@ export default function Page() {
       logoUri: formData.logoUri,
     });
     setIsLoadingIpfs(false);
-    console.log("~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~")
-    console.log(`~*~*~*~*~*~*~*~*~*~*~*~*~*~ QUOTE ${format(new Date(), "HH:mm:ss a")} ~*~*~*~*~*~*~*~*~*~*~*~*~*~`);
-    console.log("~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~*~")
-    const writeData = formData.chainIds.map(chainId => {
-      // returns empty deployer config until new suckers are deployed
-      const suckerDeployerConfig = parseSuckerDeployerConfig();
 
-      try {
-        const deployData = parseDeployData(formData, {
-          metadataCid,
-          chainId,
-          suckerDeployerConfig: suckerDeployerConfig,
-        });
+    if (!address) return;
+    const suckerDeployerConfig = parseSuckerDeployerConfig();
 
-        const encodedData = encodeFunctionData({
-          abi: revDeployerAbi, // ABI of the contract
-          functionName: "deployFor",
-          args: deployData,
-        });
+    /**
+     *  1. Build revnet data
+     */
+    const deployData = parseDeployData(formData, {
+      metadataCid,
+      chainId: sepolia.id,
+      suckerDeployerConfig: suckerDeployerConfig,
+    });
+    const encodedData = encodeFunctionData({
+      abi: revDeployerAbi, // ABI of the contract
+      functionName: "deployFor",
+      args: deployData,
+    });
 
-        return {
-          data: encodedData,
-          chain: Number(chainId),
-          deployer: revDeployerAddress[chainId],
-        }
-      } catch (e: any) {
-        toast({
-          variant: "destructive",
-          title: "Error",
-          description:
-            e.message ||
-            "Error encoding transaction",
-        });
-        throw e;
-      }
+    /**
+     *  2. Sign a ForwardRequest
+     */
+    const deadline = Date.now() + 3600 * 48 * 1000;
+    const nonce = await readErc2771ForwarderNonces(config, {
```

Comment:
Nonce is coming back as 0... this is per etherscan too

<img width="317" alt="Screenshot 2025-02-17 at 7 54 46 AM" src="https://github.com/user-attachments/assets/209e7ab0-5645-438d-85f9-237d52d8995e" />